src/pages/instances/MigrateInstanceModal.tsx

```diff
@@ -1,197 +1,100 @@
+  const [target, setTarget] = useState("");
 
   const handleEscKey = (e: KeyboardEvent<HTMLElement>) => {
     if (e.key === "Escape") {
       close();
     }
   };
 
-  const handleMigrate = (instance: LxdInstance) => {
-    migrate(selectedMember, instance);
-    close();
-  };
-
-  const handleCancel = () => {
-    if (selectedMember) {
-      setSelectedMember("");
-      return;
-    }
-
-    close();
-  };
-
-  const headers = [
-    { content: "Name", sortKey: "name" },
-    { content: "Roles", sortKey: "roles" },
-    { content: "Architecture", sortKey: "architecture" },
-    { content: "Status", sortKey: "status" },
-    { "aria-label": "Actions", className: "actions" },
-  ];
-
-  const rows = members.map((member) => {
-    const disableReason =
-      member.server_name === instance.location
-        ? "Instance already running on this member"
-        : "";
-
-    const selectMember = () => {
-      if (disableReason) {
-        return;
-      }
-
-      setSelectedMember(member.server_name);
-    };
-
-    return {
-      className: "u-row",
-      columns: [
-        {
-          content: (
-            <div
-              className="u-truncate migrate-instance-name"
-              title={member.server_name}
-            >
-              {member.server_name}
-            </div>
-          ),
-          role: "cell",
-          "aria-label": "Name",
-          onClick: selectMember,
-        },
-        {
-          content: member.roles.join(", "),
-          role: "cell",
-          "aria-label": "Roles",
-          onClick: selectMember,
-        },
-        {
-          content: member.architecture,
-          role: "cell",
-          "aria-label": "Architecture",
-          onClick: selectMember,
-        },
-        {
-          content: member.status,
-          role: "cell",
-          "aria-label": "Status",
-          onClick: selectMember,
-        },
-        {
-          content: (
-            <Button
-              onClick={selectMember}
-              dense
-              title={disableReason}
-              disabled={Boolean(disableReason)}
-            >
-              Select
-            </Button>
-          ),
-          role: "cell",
-          "aria-label": "Actions",
-          className: "u-align--right",
-          onClick: selectMember,
-        },
-      ],
-      sortData: {
-        name: member.server_name.toLowerCase(),
-        roles: member.roles.join(", ").toLowerCase(),
-        archietecture: member.architecture.toLowerCase(),
-        status: member.status.toLowerCase(),
-      },
-    };
-  });
-
-  if (isLoading) {
-    return <Loader />;
+  let modalTitle: ReactNode = isClustered
+    ? `Choose migration method for instance ${instance.name}`
+    : `Choose the target storage pool for instance ${instance.name}`;
+
+  const migrationEntity = isClustered
+    ? type?.split("-").join(" ")
+    : "storage pool";
+
+  if (["cluster-member", "storage-pool"].includes(type || "")) {
```

Comment:
```suggestion
  if (type !== "") {
```

Reply:
thanks this is much better! Just need to adjust the `MigrationType` type to include "" as well.